sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/commands.scala

```diff
@@ -47,18 +47,38 @@ case class AnalyzeTable(tableName: String) extends RunnableCommand {
 }
 
 /**
- * Drops a table from the metastore and removes it if it is cached.
+ * Drops a table/view from the metastore and removes it if it is cached.
  */
 private[hive]
 case class DropTable(
```

Comment:
Actually, I tried it. : ) That is why I submitted the PR https://github.com/apache/spark/pull/12134. 

A little bit hard. Let me list what I hit:
- unable to recognize the exception `org.apache.hadoop.hive.ql.metadata.InvalidTableException`, if we move it to sql/core. 
- some operations are hive only. For example, `invalidateTable`, which is to remove the table from the `metastoreCatalog`
- SQLContext does not allow `Drop Table`.

I did not do it since I assume we will combine Hive Context and SQL Context very soon. At that moment, it might be much easier?